client/components/app-promo/index.jsx

```diff
@@ -2,20 +2,22 @@
  * External dependencies
  */
 import React from 'react';
+import { connect } from 'react-redux';
 import Gridicon from 'components/gridicon';
 
-import analytics from 'lib/analytics';
+import { recordTracksEvent } from 'state/analytics/actions';
 import store from 'store';
 import userUtils from 'lib/user/utils';
+import { isChromeOS } from 'lib/user-agent-utils';
 import viewport from 'lib/viewport';
+import { translate } from 'i18n-calypso';
```

Comment:
We will want to use the `localize` wrapper instead of using `translate` directly. That puts `translate` in as a prop to the component. We can then use a default prop of `identity` from **lodash** for testing.

```js
…
render() {
	const { translate } = this.props;

	…
}
…

export default connect( … )( localize( AppPromo ) );
```